chapter_0/Readme.md

```diff
@@ -0,0 +1,30 @@
+####My terminal outputs
+
+`(104389484_chapter_0_master) Lucas Henderson
+Hutchs-Mac-mini:chapter_0 $ ruby -v
+ruby 2.2.3p173 (2015-08-18 revision 51636) [x86_64-darwin14]`
+
+`(104389484_chapter_0_master) Lucas Henderson
+Hutchs-Mac-mini:chapter_0 $ mkdir mystuff`
+
+`(104389484_chapter_0_master) Lucas Henderson
+Hutchs-Mac-mini:chapter_0 $ cd mystuff`
+
+`(104389484_chapter_0_master) Lucas Henderson
+Hutchs-Mac-mini:mystuff $ ls`
+
+`(104389484_chapter_0_master) Lucas Henderson
+ Hutchs-Mac-mini:mystuff $ ls
+ test.md`
+ 
+`(104389484_chapter_0_master) Lucas Henderson
+ Hutchs-Mac-mini:mystuff $`
+ 
+####Links
+ 
+ *  Array - https://rubymonk.com/learning/books/1-ruby-primer/chapters/1-arrays/lessons/2-arrays-introduction
```

Comment:
Please link to the official Ruby documentation, i.e. `http://ruby-doc.org/core-2.2.3/Array.html`